src/main/java/frc/robot/commands/ManualElevatorControl.java

```diff
@@ -0,0 +1,36 @@
+//TODO: Need to figure out a way to ovverride the constant PID adjustment happening in Elevator.java subsystem to run this command
+
+
+package frc.robot.commands;
+
+import frc.robot.controls.Operator;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Elevator;
+
+public class ManualElevatorControl extends CommandBase {
+  Elevator m_elevator; 
+
+  public ManualElevatorControl(Elevator elevator) {
```

Comment:
For this command and others, add a javadoc comment above the constructor explaining simply what the command does.

Reply:
Javadoc comments have been added.